src/anubhav/scannerexamples/ShopScanner.java

```diff
@@ -0,0 +1,156 @@
+package anubhav.scannerexamples;
+
+import java.util.Scanner;
+
+/*1. Create a system (Shop) which maintains each item's quantity.
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala’s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, 
+	print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. 
+	Partial selling is allowed.  
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.*/
+
+public class ShopScanner {
+	
+	int maggiQuantity; int dosaQuantity; int oilPouchQuantity; int panipuriQuantity; int masalaQuantity;
+	int maggiPicked; int dosaPicked; int oilPouchPicked; int panipuriPicked; int masalaPicked;
+	
+	void quantityPicked (){
+		
+		if (maggiPicked>0){
+			maggiQuantity= maggiQuantity-maggiPicked;
+			if (maggiQuantity<0){
+				System.out.println("Insufficient maggi stock");
```

Comment:
you should print invalid input instead of Insufficient maggi because we have maggi but due to invalid input we cannot process further.